src/validation.cpp

```diff
@@ -60,165 +60,27 @@
 #define MICRO 0.000001
 #define MILLI 0.001
 
-/**
- * Global state
- */
-namespace {
-    struct CBlockIndexWorkComparator
-    {
-        bool operator()(const CBlockIndex *pa, const CBlockIndex *pb) const {
-            // First sort by most total work, ...
```

Comment:
git-style-nit in commit d7c97edeea move-only: make the CChainState interface public


Could keep the previous indentation to preserve the `git blame` and minimize the diff line count? No strong opinion, though.
Please don't address my nit if you don't have to touch the patches for other reasons.

Reply:
I don't know that we should maintain a weird indentation just for ease-of-diff, but if other reviewers disagree, I'll make the change.

Reply:
I don't mind the indentation `git show --color-moved=dimmed_zebra --color-moved-ws=ignore-all-space d7c97edeea8cee10ad9da1f940d39d5073ac142d`.

I think this operator was inlined? Does it make sense to declare it `inline`?